src/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Xml/X509IssuerSerial.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography.Xml
```

Comment:
Ugh, hopefully this doesn't cause us a layering problem later.  Maybe we want to move it to X509 or Primitives now to avoid drama later?